internal/set/set.go

```diff
@@ -0,0 +1,91 @@
+/*
+Copyright 2021 Flant JSC
```

Comment:
we sure about 2021?

Reply:
This is a DH file from an existing linter. The year of creation does not change when reused.